script.js

```diff
@@ -1,4 +1,943 @@
+  var randomInteger = Math.floor(randomDecimal);
+  return randomInteger + 1;
+};
+
+//GLOBAL Definitions.
+var myOutputValue = "";
+var diceNumberRolled = 0;
+var numberOfPlayers = 0;
+var playerIndexNmbr = 0;
+var playerName = "";
+var gameCount = 0;
+var currentPlayerSumOfNumbers = 0;
+var gameLogic = "highest";
+var gameLogicDisplay = "";
+var qtyOfDice = 0;
+var leaderBoardArray = [];
+
+//Game Logic and Service parts
+
+var getNumberOfPlayers = function (qtyOfPlayers) {
+  // use over arcing "IF /ELSE " to control for highest or lowest mode and return error messages
+  if (qtyOfPlayers > 1 && isNaN(qtyOfPlayers) == false) {
+    numberOfPlayers = qtyOfPlayers;
+    var message0 = `Number of Players = ${qtyOfPlayers}
+                  <br><br>Please input how many dices to use: `;
+    gameMode = `inputQtyOfDice`;
+  } else {
+    message0 = `xxxx INPUT ERROR xxxx :<br>Please key in only numerals (> 1)`;
+    gameMode = "getNumberOfPlayers";
+  }
+
+  gameLogicDisplay = ` Game Logic: ${gameLogic.toUpperCase()} combined number wins<br><br>`;
+
+  return gameLogicDisplay + message0;
+};
+
+var inputQtyOfDice = function (numberOfDice) {
+  qtyOfDice = numberOfDice; // use over arcing "IF /ELSE " to control for highest or lowest mode and return error messages
+
+  // Message Output Definition
+  var message1 = `All players have agreed to have  ${numberOfDice} dices rolled for this round.
+                  <br><br>Please type "knock out" or "multiplayer" in the input to determine whether players play in knockout fashion or all at once (multiplayer).
+                  <br><br>Click 'Submit' to proceed when ready.
+                  `;
+
+  // <br><br>Please hit submit again to generate the ${gameLogic.toUpperCase()} combined number
+  // <br> Or key in "lowest" or "highest" to switch game logic`;
+  gameMode = `setMultiplayerOrKnockOut`;
+
+  gameLogicDisplay = ` Game Logic: ${gameLogic.toUpperCase()} combined number wins<br><br>`;
+
+  return gameLogicDisplay + message1;
+};
+
+var setMultiplayerOrKnockOut = function (playMode) {
+  var displayPlayMode = "";
+  var message2 = "";
+  var errorMessage = `XXX INPUT ERROR XXX
+                      <br>Please input 'knock out' or 'multiplayer' only.`;
+
+  if (playMode.toLowerCase() == "knock out") {
+    gameMode = `arrangeDiceKnockOut`;
+    displayPlayMode = "Knock Out";
+    message2 = `Game will be played in ${displayPlayMode.toUpperCase()} style.
+                 <br>Please click "Submit" to proceed.`;
+  } else if (playMode.toLowerCase() == "multiplayer") {
+    gameMode = `arrangeDiceMultiplayer`;
+    displayPlayMode = "Multiplayer";
+    message2 = `Game will be played in ${displayPlayMode.toUpperCase()} style.
+                 <br>Please click "Submit" to proceed.`;
+  } else {
+    gameMode = `setMultiplayerOrKnockOut`;
+    message2 = errorMessage;
+  }
+
+  gameLogicDisplay = ` Game Logic: ${gameLogic.toUpperCase()} combined number wins<br><br>`;
```

Comment:
I think what you could do help cut down on some lines is to do the entire if else if else block and then define your message2 after that. Since message2 only changes with the name of the play mode, then you pl;ug the name of the variable afterwards. You can just return the errorMessage  on line 643 so that this situation works incase of an error